src/pages/Login/Login.js

```diff
@@ -1,8 +1,98 @@
-import React from 'react';
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 import './Login.scss';
 
 const Login = () => {
-  return <div>Login</div>;
+  const [loginValue, setLoginValue] = useState({
+    id: '',
+    pw: '',
+  });
+
+  const navigate = useNavigate();
+  const onLogin = () => {
+    const valid = loginValue.id === '';
+
+    if (loginValue.id === '' || loginValue.pw === '') {
+      valid
+        ? alert('[ 아이디 ] 는(은) 필수항목입니다.')
+        : alert('[ 비밀번호 ] 는(은) 필수항목입니다.');
```

Comment:
```suggestion
const onLogin = () => {
    const emptyIdInput = loginValue.id.trim() === '';
    const emptyPasswordInput = loginValue.pw.trim() === '';
    const emptyLoginInput = emptyIdInput && emptyPasswordInput;
    if (emptyIdInput) {
      alert('[ 아이디 ] 는(은) 필수항목입니다.');
    } else if (emptyPasswordInput) {
      alert('[ 비밀번호 ] 는(은) 필수항목입니다.');
    }

    if (!emptyLoginInput) {
  
```

* 간단한 분기처리인데, 조건문에 삼항연산자가 중첩되어있어서 간단하게 if문으로만 써도 될것같습니다.
* 변수명도 어떤 역할을 하는지 를 생각해서 이름 지어 주면, 좀더 추상적이지 않고 직관적으로 이해할 수 있습니다.
* .trim() 이 무엇인지 찾아보세요.

Reply:
넵 수정하고 확인했습니다